<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">


   <widget class="QLineEdit" name="input_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>

   <widget class="QPushButton" name="button_clear">
    <property name="geometry">
        <rect>
            <x>190</x>
            <y>20</y>
            <width>50</width>
            <height>40</height>
        </rect>
    </property>
    <property name="text">
        <string>Clear</string>
    </property>
</widget>

   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
    <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>

    <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>

 <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>

   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>

    <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>

    <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
  
    <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>

     <widget class="QPushButton" name="button_multiply">
    <property name="geometry">
        <rect>
            <x>80</x>
            <y>150</y>
            <width>50</width>
            <height>40</height>
        </rect>
    </property>
    <property name="text">
        <string>*</string>
    </property>
</widget>
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Input:</string>
    </property>
   </widget>

   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
    <widget class="QLabel" name="postfix_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Postfix Expression:</string>
    </property>
    <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>

   </widget>
       <widget class="QLabel" name="postfix_display">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>

    <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>

   </widget>

       <widget class="QLabel" name="prefix_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefix Expression:</string>
    </property>
    <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>

   </widget>


       <widget class="QLabel" name="prefix_display">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>

    <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>

   </widget>

   <widget class="QLCDNumber" name="output_lcd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>9</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_equal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>